projects/igniteui-angular/src/lib/grids/grid/grid-row-editing.spec.ts

```diff
@@ -1450,62 +1395,59 @@ describe('IgxGrid - Row Editing #grid', () => {
             expect(cell.editMode).toBeFalsy();
         }));
 
-        it(`Hiding: Should exit edit mode when hiding a column`, fakeAsync(() => {
+        it(`Hiding: Should exit edit mode when hiding a column`, () => {
             const gridAPI: IgxGridAPIService = (<any>grid).gridAPI;
 
-            const targetCell = grid.getCellByColumn(0, 'ProductName'); // Cell must be editable
-            targetCell.setEditMode(true);
-            tick(16);
+            cell.setEditMode(true);
+
             fix.detectChanges();
             expect(gridAPI.get_cell_inEditMode()).toBeTruthy(); // check if there is cell in edit mode
             spyOn(gridAPI, 'escape_editMode').and.callThrough();
 
-            targetCell.column.hidden = true;
-            tick(16);
+            cell.column.hidden = true;
+
             fix.detectChanges();
 
             expect(gridAPI.escape_editMode).toHaveBeenCalled();
             expect(grid.rowEditingOverlay.collapsed).toBeTruthy();
-        }));
+        });
 
-        it(`Hiding: Should show correct value when showing the column again`, fakeAsync(() => {
+        it(`Hiding: Should show correct value when showing the column again`, () => {
             grid.showToolbar = true;
             grid.columnHiding = true;
-            tick(16);
+
             fix.detectChanges();
 
             const targetCbText = 'Product Name';
-            const targetCell = grid.getCellByColumn(0, 'ProductName');
-            targetCell.setEditMode(true);
-            tick(16);
-            targetCell.update('Tea');
+            cell.setEditMode(true);
+
+            cell.update('Tea');
 
             // hide column
             grid.toolbar.columnHidingButton.nativeElement.click();
-            tick(16);
+
             const overlay = fix.debugElement.query(By.css('.igx-column-hiding__columns'));
             const checkboxes = overlay.queryAll(By.css('.igx-checkbox__label'));
             const targetCheckbox = checkboxes.find(el => el.nativeElement.innerText.trim() === targetCbText);
             targetCheckbox.nativeElement.click();
-            tick(16);
+
             // show column
             targetCheckbox.nativeElement.click();
-            tick(16);
```

Comment:
This can be changed with:
   const columnChooser = GridFunctions.getColumnHidingElement(fix);
            GridFunctions.clickColumnChooserItem(columnChooser, targetCbText);
            GridFunctions.clickColumnChooserItem(columnChooser, targetCbText);